examples/large_memory_tests/test_bicubic.py

```diff
@@ -0,0 +1,50 @@
+#
```

Comment:
I think I understand why you excluded the examples from the test directories. 

You should rename this file to `large_memory_checks/check_bicubic.py` to escape pytest.

Reply:
Exactly. I just did that.